include/z64animation.h

```diff
@@ -1,5 +1,5 @@
-#ifndef Z64_ANIMATION_H
-#define Z64_ANIMATION_H
+#ifndef Z64ANIMATION_H
+#define Z64ANIMATION_H
```

Comment:
Personally, I prefer as it was, with the underscore after the `Z64_`. I find it more readable that way. Is there an underlying reason for one way or the other?

Reply:
If I had to guess it would be because I have made a bunch of PRs without that underscore. I don't have a hard preference about this so I could change my PRs to have the underscore if that's preferred

Reply:
2 reasons:
1. There seemed to be more already in this format.
2. The bigger reason is it matches the file name better which has that format.

I'm not opposed to changing it (I agree it does look a little better), but probably should change the file names as well imo.

Reply:
I suppose it's fine as you've changed it then